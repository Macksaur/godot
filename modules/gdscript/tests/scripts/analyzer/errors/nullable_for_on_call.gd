func test():
    for element in nullable_array(): # Should error
        print(element)

func nullable_array() -> Array?:
    return null
