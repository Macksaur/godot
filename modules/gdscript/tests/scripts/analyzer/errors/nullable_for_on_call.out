GDTEST_ANALYZER_ERROR
Potentially unsafe for the nullable return of "nullable_array()" to be used in a for-loop.
